do/serverless.go

```diff
@@ -678,6 +681,42 @@ func (s *serverlessService) DeleteNamespace(ctx context.Context, name string) er
 	return err
 }
 
+func (s *serverlessService) CleanNamespace() error {
+	// Deletes all triggers
+	ctx := context.TODO()
+	triggers, err := s.ListTriggers(ctx, "")
+
+	if err == nil {
+		for _, trig := range triggers {
+			err = s.DeleteTrigger(ctx, trig.Name)
+			if err != nil {
+				return err
+			}
+		}
+	}
+
+	// Deletes all functions
+	fns, err := s.ListFunctions("", 0, 0)
+	if err != nil {
+		return err
+	}
+
+	for _, fn := range fns {
+		name := fn.Name
+		pkg := strings.Split(fn.Namespace, "/")
+		if len(pkg) == 2 {
+			name = pkg[1] + "/" + fn.Name
+		}
+
+		err := s.DeleteFunction(name, false)
```

Comment:
I understand why you're using `false` here; a clarifying comment might be in order though.